### PR TITLE
*WIP* Final Improvements

### PR DESCRIPTION
**Pull request by mojowen**
- [x] Break maps into separate ajax calls
- [ ] Load maps from div tags
- [ ] Load styles asynchronously
- [ ] support multiple maps per page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <html>
   <head>
   <meta charset="utf-8">
-  <link rel="stylesheet" href="styles/style.css">
+  <link rel="stylesheet" href="styles/style.scss">
```
Comment (mojowen) [inline index.html]: just realized we're going to need to asynchronously add this too

File: package.json [hunk 2]
```diff
@@ -32,6 +32,11 @@
   "dependencies": {
     "d3": "^5.9.2",
     "marked": "^0.6.1",
+    "parcel-plugin-static-files-copy": "^2.0.0",
     "topojson": "^3.0.2"
-  }
+  },
+    "staticFiles": {
```
Comment (mojowen) [inline package.json]: this allows serving static files

File: src/constants.js [hunk 3]
```diff
@@ -13,3 +13,7 @@ export const labelMap = {
   house: "House district"
 };
 export const prefix = "dfp-map__";
+export const rootURL =
+  process.env.NODE_ENV === "development"
```
Comment (mojowen) [inline src/constants.js]: not completely sure this is no set to develop for github pages

File: package.json (hunk 2 above)
Comment (noahmanger) [reply package.json]: Ah nice.

**Pull request merged**